woocommerce/woocommerce-ios
# Store creation: update copy for progress steps when waiting for the store to be ready

Closes: #10045 


## Description


Why we're doing this: p1687358531678699-slack-C03L1NF1EA3

This PR updated the copy for each progress step (6 in total). The subtitle has different bolded text for each progress step, and the bolded text is now implemented with markdown in the SwiftUI `Text`.

To allow easier SwiftUI previews for all the progress steps, the initial progress step is now DI'ed to the view model with the same default value.

## Testing instructions


- Log in with WPCOM if needed
- Go to the Menu tab
- Tap on the store name to switch store, then scroll down to the bottom and tap `Add a Store`
- Tap `Create a new store`
- Continue to all the steps to create a store --> at the end, the progress view should show the expected copy as in the latest design VyLr7LvKodHE4kINfBE7Lw-fi-4468%3A113292

## Screenshots


step | light / dark
-- | --
step 1 | <img width="818" alt="Screenshot 2023-06-21 at 3 42 07 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/fa035934-e04f-40a5-8f4e-ed94d2d4d932">
step 2 | <img width="802" alt="Screenshot 2023-06-21 at 3 42 21 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/9fb0bf9e-c6f7-420d-a7f5-b3e7f5039dc5">
step 3 | <img width="808" alt="Screenshot 2023-06-21 at 3 42 29 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/6d5ea8c5-dedc-439b-849f-39176c5dd284">
step 4 | <img width="804" alt="Screenshot 2023-06-21 at 3 42 42 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/084e3174-3188-4c3a-938d-195ca6414717">
step 5 | <img width="798" alt="Screenshot 2023-06-21 at 3 42 57 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/d8ce0abd-e449-4704-ab08-4b65193e70a0">
step 6 | <img width="806" alt="Screenshot 2023-06-21 at 3 43 11 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/1a307d00-c8be-42f0-b671-c61b48783e96">


---
